packages/apt/install_test.sh

```diff
@@ -5,14 +5,31 @@ set -exu
 package=$1
 
 sudo apt update
-sudo apt install -y -V software-properties-common lsb-release
-sudo add-apt-repository -y universe
-sudo add-apt-repository \
-     "deb http://security.ubuntu.com/ubuntu $(lsb_release --short --codename)-security main restricted"
+sudo apt install -V -y lsb-release
+
+distribution=$(lsb_release --id --short | tr 'A-Z' 'a-z')
+code_name=$(lsb_release --codename --short)
+
+case ${distribution} in
+  debian)
+    sudo apt install -y -V wget
+    wget https://apache.jfrog.io/artifactory/arrow/debian/apache-arrow-apt-source-latest-${code_name}.deb
+    sudo apt install -y -V ./apache-arrow-apt-source-latest-${code_name}.deb
+    wget https://packages.groonga.org/debian/groonga-apt-source-latest-${code_name}.deb
+    sudo apt install -y -V ./groonga-apt-source-latest-${code_name}.deb
+  ;;
```

Comment:
Should we enable shfmt pre-commit configuration https://github.com/scop/pre-commit-shfmt ?

```suggestion
    ;;
```